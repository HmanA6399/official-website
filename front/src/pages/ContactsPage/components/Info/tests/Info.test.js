import React from 'react';
import Info from '../index';

import { create } from "react-test-renderer";

// because it's static component and doesn't change often
it('matches snapshot', () => {
  const component = create(<Info />);

  expect(component.toJSON()).toMatchSnapshot()
});