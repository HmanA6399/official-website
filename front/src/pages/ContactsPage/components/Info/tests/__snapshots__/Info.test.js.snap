// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<article
  className="bg-section"
  id="contact-info"
>
  <section
    id="info"
  >
    <div>
      <h4>
         ADDRESS
      </h4>
      <p>
        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nostrum, voluptate?
      </p>
    </div>
    <div>
      <h4>
         EMAIL
      </h4>
      <p>
        example@domain.com
      </p>
    </div>
    <div>
      <h4>
         PHONE
      </h4>
      <p>
        +201223456789
      </p>
    </div>
  </section>
  <section
    id="map"
  >
    <h2>
      OUR LOCATION
    </h2>
    <img
      alt="map"
      src="placeholder.png"
    />
  </section>
</article>
`;
